python/ray/experimental/client/common.py

```diff
@@ -17,83 +18,230 @@ def __repr__(self):
     def __eq__(self, other):
         return self.id == other.id
 
+    def binary(self):
+        return self.id
+
 
 class ClientObjectRef(ClientBaseRef):
     pass
 
 
-class ClientActorRef(ClientBaseRef):
+class ClientActorNameRef(ClientBaseRef):
+    pass
+
+
+class ClientStub:
     pass
 
 
-class ClientRemoteFunc:
+class ClientRemoteFunc(ClientStub):
+    """
+    A stub created on the Ray Client to represent a remote
+    function that can be exectued on the cluster.
+
+    This class is allowed to be passed around between remote functions.
+
+    Args:
+        _func: The actual function to execute remotely
+        _name: The original name of the function
+        _ref: The ClientObjectRef of the pickled code of the function, _func
```

Comment:
We should change this to use a strategy similar to handle remote actors (pickling the method handle and returning it). If you don't want to do it in this PR, can you add a TODO to make it consistent?

Reply:
Yeah, that requires the object pinning fix we talked about with Stephanie though. So yeah, TODO